modules/apps/static/portal-k8s-agent/portal-k8s-agent-impl/src/main/java/com/liferay/portal/k8s/agent/internal/osgi/commands/ClientExtensionsOSGiCommands.java

```diff
@@ -38,79 +39,82 @@
 @Component(
 	property = {
 		"osgi.command.function=list", "osgi.command.function=reload",
-		"osgi.command.function=show", "osgi.command.scope=cxconfig"
+		"osgi.command.function=show", "osgi.command.scope=clientextensions"
 	},
 	service = OSGiCommands.class
 )
-public class CXConfigOSGiCommands implements OSGiCommands {
+public class ClientExtensionsOSGiCommands implements OSGiCommands {
 
 	public void list(String... filters)
 		throws InvalidSyntaxException, IOException, PortalException {
 
-		Configuration[] cxConfigurations = _getConfigurations(filters);
+		Configuration[] configurations = _getConfigurations(filters);
 
-		if (ArrayUtil.isEmpty(cxConfigurations)) {
-			System.out.println("No configurations found.");
+		if (ArrayUtil.isEmpty(configurations)) {
+			System.out.println(
+				"Could not find configuration for filters " +
+					Arrays.toString(filters));
 
 			return;
 		}
 
-		_printConfigurations(cxConfigurations);
+		_printConfigurations(configurations);
 	}
 
-	public void reload(String... args)
+	public void reload(String... arguments)
```

Comment:
Take a look at the other `*OSGiCommands.java` classes. They all use typed and named parameters for their command methods. `UpgradeOSGiCommands` and `LoggingOSGiCommands` are good examples. 

Use specific parameters for each of these command methods. I think `list` is ok (except for the recommended rename), but `reload` and `show` both just expect one argument (the `pid`), so you can just put that directly in the method signature. 

Once you do, test what happens when you pass in fewer or more arguments (does `gogo` do the validation for you?), then update the tests accordingly if needed.